CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - ESP32: fix i2c_driver_acquire and i2c_driver_release functions, that were working only once.
+- Sending messages to registered processes using the `!` operator now works.
+- Fixed bug in `OP_SEND` that would accept sending a message to any integer.
```

Comment:
Te situation was a bit worse: we did allow send to any term without raising any error, and for integers was even working.

Reply:
I updated the explanation of the bug.